docs/manual/src/udl/ext_types_wrapped.md

```diff
@@ -125,15 +124,40 @@ The behavior of the generated scaffolding will be:
 * Calling `take_handle_2` with a value of `-1` will always panic.
 * All other values will return `Ok(ExampleHandle)`
 
-## Custom Foreign Wrappers
+## Custom types in the bindings code
+
+*Note: The facility described in this document is not yet available for the Ruby bindings.*
+
+By default, the foreign bindings just see the builtin type - eg, the bindings will get an integer
+for the `Handle`.
```

Comment:
Perhaps use a different example here, suggest URL or UUID.

I got confused because `Handle` is not customized in the `examples/` but is in this documentation.